Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1292,6 +1296,56 @@
     <!-- XTXT: Description in statistics card that is displayed below the incidence value-->
     <string name="statistics_card_incidence_value_description">bestätigte Neuinfektionen je 100.000 Einwohner</string>
 
+    <!-- Explanation screen -->
+    <!-- XHED: Explanation screen title -->
+    <string name="statistics_explanation_title">Kennzahlen</string>
```

Comment:
Please use "" for all the strings e.g. `<string name="statistics_explanation_title">"Kennzahlen"</string>`